tests/Fixtures/e2e/Test_Frameworks/README.md

```diff
@@ -0,0 +1,9 @@
+# Test Framework Equality
```

Comment:
is this README necessary?

Reply:
Not 100%, but it makes it clear what the test is trying to prove